scripts/hist_importer.py

```diff
@@ -135,7 +135,9 @@ def insert_qb(history, dest):
         'INSERT INTO History (url,title,atime,redirect) VALUES (?,?,?,?)',
         history
     )
-    cursor.execute('DROP TABLE CompletionHistory')
+    cursor.execute('DELETE FROM CompletionHistory')
```

Comment:
I think this shouldn't be required at all anymore if you set `force_rebuild`, no?

Reply:
Without this im having this error when starting qb:

```
14:18:13 ERROR: Uncaught exception                                                                                         
Traceback (most recent call last):                  
  File "/usr/bin/qutebrowser", line 33, in <module>                                                                        
    sys.exit(load_entry_point('qutebrowser==2.0.2', 'gui_scripts', 'qutebrowser')())                                       
  File "/usr/lib/python3/dist-packages/qutebrowser/qutebrowser.py", line 208, in main
    return app.run(args)
  File "/usr/lib/python3/dist-packages/qutebrowser/app.py", line 128, in run                                               
    init(args=args)           
  File "/usr/lib/python3/dist-packages/qutebrowser/app.py", line 157, in init                                             
    _init_modules(args=args)                                                                                               
  File "/usr/lib/python3/dist-packages/qutebrowser/app.py", line 486, in _init_modules                                     
    history.init(objects.qapp)                                                                                             
  File "/usr/lib/python3/dist-packages/qutebrowser/browser/history.py", line 465, in init       
    web_history = WebHistory(progress=progress, parent=parent)                                                            
  File "/usr/lib/python3/dist-packages/qutebrowser/browser/history.py", line 191, in __init__                             
    self._rebuild_completion()                               
  File "/usr/lib/python3/dist-packages/qutebrowser/browser/history.py", line 278, in _rebuild_completion                   
    self.completion.delete_all()
  File "/usr/lib/python3/dist-packages/qutebrowser/misc/sql.py", line 459, in delete_all
    Query("DELETE FROM {table}".format(table=self._name)).run()
  File "/usr/lib/python3/dist-packages/qutebrowser/misc/sql.py", line 243, in __init__
    self._check_ok('prepare', ok)
  File "/usr/lib/python3/dist-packages/qutebrowser/misc/sql.py", line 264, in _check_ok
    raise_sqlite_error(msg, error)
  File "/usr/lib/python3/dist-packages/qutebrowser/misc/sql.py", line 168, in raise_sqlite_error
    raise BugError(msg, error) 
qutebrowser.misc.sql.BugError: Failed to prepare query "DELETE FROM CompletionHistory": "no such table: CompletionHistory U
nable to execute statement"
14:18:13 ERROR: Error while getting history: 'command-history'
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/qutebrowser/misc/crashsignal.py", line 214, in _get_exception_info
File "/usr/lib/python3/dist-packages/qutebrowser/misc/crashsignal.py", line 214, in _get_exception_info
    cmd_history = objreg.get('command-history')[-5:]
  File "/usr/lib/python3/dist-packages/qutebrowser/utils/objreg.py", line 247, in get
    return reg[name]
  File "/usr/lib/python3.9/collections/__init__.py", line 1058, in __getitem__
    raise KeyError(key)
KeyError: 'command-history'
```

Reply:
But that's with the old line readded, no? What I meant is to remove it completely, i.e. only set force_rebuild.

Reply:
Yes its with the old line ah ok the force rebuild take care of everything I get it I will update the patch soon :tm:

Reply:
I've amend the commit 💚